server/routes/adjustmentRoutes.ts

```diff
@@ -51,7 +53,12 @@ export default class AdjustmentRoutes {
 
     const message = req.flash('message')
     const messageExists = message && message[0]
-    if (messageExists) {
+    const reviewDeductions = this.paramStoreService.get(req, 'returnToReviewDeductions')
+    if (reviewDeductions) {
+      this.paramStoreService.clear(req, 'returnToReviewDeductions')
+    }
```

Comment:
I think you also need to clear the adjustments store service on visiting the hub. Or on first entry into review unused deductions.

If you start creating some remand/tagged bail and then enter review unused deductions without actually creating the adjustment, the page gets very confused.

Reply:
This is already being done.
See: 
- `adjustmentRoutes.ts` lines 56-63
- `unusedDeductionRoutes.ts` lines 204-209